cpp/include/cudf/reduction.hpp

```diff
@@ -55,18 +58,22 @@ namespace cudf {
  *
  * @param[in] col Input column
  * @param[in] op  The operator applied by the reduction
- * @param[in] dtype The computation and output precision.
+ * @param[in] output_dtype  The computation and output precision.
  *     `dtype` must be a data type that is convertible from the input dtype.
- *     If the input column has arithmetic type, any arithmetic type can be specified.
+ *     If the input column has arithmetic type or cudf::bool8 type,
+ *     output_dtype can be any arithmetic type or cudf::bool8 type.
+ *     For `mean`, `var` and `std` ops, a floating point type must be specified.
  *     If the input column has non-arithmetic type
  *     (date32, timestamp, category...), the same type must be specified.
+ * @param[in] ddof Delta Degrees of Freedom used for `std`, `var`
  *
  * @returns  gdf_scalar the result value
  * If the reduction fails, the member is_valid of the output gdf_scalar
  * will contain `false`.
  * ----------------------------------------------------------------------------**/
 gdf_scalar reduction(const gdf_column *col, gdf_reduction_op op,
-                        gdf_dtype output_dtype);
+                        gdf_dtype output_dtype,
+                        gdf_size_type ddof = 1);
```

Comment:
defaulting of this parameter is done because reduction ops other than `var` and `std`, does not use this parameter.

Reply:
Good point. I

Reply:
Should the default be `1` or `0`?

Reply:
Pandas defaults to 1 which is why I assumed it was defaulted to 1 here. Not sure what the right call is. As a user unfamiliar with Pandas, I'd be surprised to try and compute a stddev to find the results are computed using `n-1` instead of `n`.